tensorflow_probability/python/internal/test_util.py

```diff
@@ -154,7 +147,7 @@ def _one_part(*structure):
     if overall_exception:
       final_msg += str(overall_exception)
       raise AssertionError(final_msg)
-    elif exceptions_with_paths:
+    if exceptions_with_paths:
```

Comment:
Not needed?

Reply:
Pylint was not happy with 'elif after raise', here some details https://stackoverflow.com/a/55632864